### ISSUE TITLE
Importing a GPX with images does not seem to complete import

### ISSUE THREAD

**Issue by cachly** [cachly/bug-tracker#1807]
**[Original report](https://bitbucket.org/zedsaidstudio/cachly/issues/1807) by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

Summary:
Import GPX from Dropbox does not indicate Progress.

Description:
Importing a GPX file from Dropbox does not show the progress indicator or get to a completion point when a new list is created within the import sequence.

Repro:
- Select Import from Dropbox
- Select Choose Offline List
- Add a new list and select it
- Select Save Images (I don't think this is necessary)
- Click import

Results:
 - No progress is indicated
 - Screen sits at with Import grayed out

Expected Results:
- Progress indicator then completion

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

I have narrowed this down.  My theory on a new list is incorrect, this issue is related to importing images.  If I select Save Images the import gets stuck.  With this being said even if I do not select Save Images the list is not useable.  I've found I need to kill the App and restart it before I can read the offline list.

Perhaps there is something about my GPX file, however I do know I did import it with 2.0 a couple of months ago. 

Here is a link to my file of you want to try it:
https://www.dropbox.com/s/jx3o9bg54s8jie3/caches2find.gpx?dl=1
